ereolen_support/ereolen_support.info

```diff
@@ -0,0 +1,6 @@
+name = eReolen support
```

Comment:
Maybe `eReolen support (jira)`

Reply:
It's not only Jira, but publizon mail support as well.